packages/yew-macro/tests/function_component_attr/hook_location-pass.rs

```diff
@@ -0,0 +1,26 @@
+use yew::prelude::*;
```

Comment:
Please add `#[no_implicit_prelude]` in passing tests. This is so macro hygiene is ensured. Macros should expand with fully qualified paths and removing the prelude helps us ensure that.